src/test/java/org/jfrog/buildinfo/integration/ArtifactoryPluginITest.java

```diff
@@ -129,6 +134,13 @@ private void testMultiModule(String projectName) throws Exception {
             assertEquals(0, CollectionUtils.size(multi3.getArtifacts()));
             assertEquals(15, CollectionUtils.size(multi3.getDependencies()));
             assertEquals(5, CollectionUtils.size(multi3.getProperties()));
+
+            Properties badProps = new Properties();
+            badProps.put("password", "password-password");
+            // Check build properties exclude
+            assertFalse(build.getProperties().contains(badProps));
+            // Check module properties exclude
+            build.getModules().forEach((value) -> assertFalse(value.getProperties().contains(badProps)));
```

Comment:
This should be in the getAndAssertBuild which is common to all integration tests